### ISSUE TITLE
google-cloud-sdk: "gcloud beta sql connect" requires cloud_sql_proxy

### ISSUE THREAD

**Issue by blueyed** [sudoforge/pkgbuilds#42]
```
% gcloud beta sql connect … --user=postgres
Starting Cloud SQL Proxy: [/opt/google-cloud-sdk/bin/cloud_sql_proxy -instances …=tcp:9470 -credential_file …/adc.json]]
ERROR: gcloud crashed (FileNotFoundError): [Errno 2] No such file or directory: '/opt/google-cloud-sdk/bin/cloud_sql_proxy'

If you would like to report this issue, please run the following command:
  gcloud feedback

To check gcloud for common problems, please run the following command:
  gcloud info --run-diagnostics
```

I've seen that it is missing from `gcloud components` and have read what to do to manually install this component in https://aur.archlinux.org/packages/google-cloud-sdk#pinned-659642, but wondered if it would be feasible to opt-depend on e.g. cloud-sql-proxy-bin, which provides the `cloud_sql_proxy` binary, and/or patch google-cloud-sdk to not require it from `/opt/google-cloud-sdk/bin/cloud_sql_proxy`, but use it via `$PATH`!?

**Comment by sudoforge** [comment]
> opt-depend on e.g. cloud-sql-proxy-bin

Unfornately, this is a no from me. That particular package has been flagged as out of date for nearly a month with no update from the author.

> and/or patch google-cloud-sdk to not require it from /opt/google-cloud-sdk/bin/cloud_sql_proxy, but use it via $PATH!?

This is fungible, but...

> and have read what to do to manually install this component in https://aur.archlinux.org/packages/google-cloud-sdk#pinned-659642

Why does this not work for you?

**Comment by sudoforge** [comment]
Hey @blueyed, can you provide feedback to the above question?

**Comment by sudoforge** [comment]
Interesting find! I'm currently traveling for a wedding, but will look into this in detail next week.

Related: I've recently finished standardizing the build process and integrating this repository with my internal development workflow, which means that I can now focus on package adoption/creation -- namely, finally get around to packaging the external components. I don't have an active timeline for you, however, I'd expect this to be the first target and should be completed roughly around the end of next week.

**Comment by blueyed** [comment]
I've noticed that `disable_updater` gets reset (likely after a package upgrade).
It would be good if it could be persisted somehow, e.g. by merging updates from a file in `/etc` maybe?

**Comment by sudoforge** [comment]
@blueyed yes, updating the package would replace that file, however, I don't think patching it by default to enable management of plugins would be ideal, as users would become used to managing plugins in that way. I got sidelined with a few other tasks that required my attention, but I will be addressing the plugins this weekend (I've carved time out in my schedule specifically for this). That should be alleviate any difficulty or repetition necessary for end users in regards to utilizing SDK plugins.
